services/fileServices/MdPageServices/__tests__/ContactUsPageService.spec.js

```diff
@@ -0,0 +1,115 @@
+const {
+  contactUsContent: mockContactUsContent,
+  contactUsSha: mockContactUsSha,
+  rawContactUsContent: mockRawContactUsContent,
+} = require("@fixtures/contactUs")
+const {
+  footerContent: mockFooterContent,
+  footerSha: mockFooterSha,
+} = require("@fixtures/footer")
+
+describe("ContactUs Page Service", () => {
+  const siteName = "test-site"
+  const accessToken = "test-token"
+  const reqDetails = { siteName, accessToken }
+
+  const CONTACT_US_FILE_NAME = "contact-us.md"
+  const CONTACT_US_DIRECTORY_NAME = "pages"
+
+  const mockFrontMatter = {
+    ...mockContactUsContent.frontMatter,
+    feedback: mockFooterContent.feedback,
+  }
+  const mockContent = mockContactUsContent.pageBody
+
+  const mockGithubService = {
+    create: jest.fn(),
+    read: jest.fn(),
+    update: jest.fn(),
+    delete: jest.fn(),
+  }
+
+  const mockFooterYmlService = {
+    read: jest.fn(),
+    update: jest.fn(),
+  }
+
+  jest.mock("@utils/markdown-utils", () => ({
+    retrieveDataFromMarkdown: jest.fn().mockReturnValue({
+      frontMatter: mockFrontMatter,
+      pageContent: mockContent,
+    }),
+    convertDataToMarkdown: jest.fn().mockReturnValue(mockRawContactUsContent),
+  }))
+
+  const {
+    ContactUsPageService,
+  } = require("@services/fileServices/MdPageServices/ContactUsPageService")
+  const service = new ContactUsPageService({
+    gitHubService: mockGithubService,
+    footerYmlService: mockFooterYmlService,
+  })
+
+  const {
+    retrieveDataFromMarkdown,
+    convertDataToMarkdown,
+  } = require("@utils/markdown-utils")
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+  mockFooterYmlService.read.mockResolvedValue({
+    content: mockFooterContent,
+    sha: mockFooterSha,
+  })
+
+  describe("Read", () => {
+    mockGithubService.read.mockResolvedValue({
+      content: mockRawContactUsContent,
+      sha: mockContactUsSha,
+    })
+    it("Reading the contact us page works correctly", async () => {
```

Comment:
could we also have sad path tests - ie, when the `read` from `gitHubService` or `footerYmlService` fails, the error propagated is correct

Reply:
Good point, I've added it in https://github.com/isomerpages/isomercms-backend/pull/349/commits/78c9173af27ab70898c3129fd88d23490291118f